openhtf/util/measurements.py

```diff
@@ -284,38 +294,34 @@ class Collection(object):  # pylint: disable=too-few-public-methods
     # [(5, 10), (6, 11)]
   """
 
-  __slots__ = ('_declarations', '_values')
+  __slots__ = ('_measurements', '_values')
 
-  def __init__(self, measurement_declarations):
+  def __init__(self, measurements):
     # We have setattr so we have to bypass it to set attributes.
-    object.__setattr__(self, '_declarations', measurement_declarations)
+    object.__setattr__(self, '_measurements', measurements)
     object.__setattr__(self, '_values', {})
 
-  def _asdict(self):
-    """Return dict of this Collection."""
-    return dict(self)
-
   def _AssertValidKey(self, name):
     """Raises if name is not a valid measurement."""
-    if name not in self._declarations:
+    if name not in self._measurements:
       raise NotAMeasurementError('Not a measurement', name)
 
   def __iter__(self):  # pylint: disable=invalid-name
-    def _GetMeasValue(item):
+    def _GetMeasurementValue(item):
       return item[0], item[1].GetValue()
-    return itertools.imap(_GetMeasValue, self._values.iteritems())
+    return itertools.imap(_GetMeasurementValue, self._values.iteritems())
 
   def __setattr__(self, name, value):
     self._AssertValidKey(name)
     record = self._values.setdefault(
-        name, MeasuredValue.ForDeclaration(self._declarations[name]))
+        name, MeasuredValue.ForMeasurement(self._measurements[name]))
     record.SetValue(value)
```

Comment:
Yep, that's the idea.  I didn't want to shift to that just yet, but it would be easy to update here later.